### PR TITLE
Hotfix for gateway user status check

### PR DESCRIPTION
**Pull request by sanyavertolet**
### What's done:
 * Added `/api/$v1/avatar/**` to `allowedForInactiveEndpoints`
 * Supported unauthorized users in gateway `WebSecurityConfig`

### CODE REVIEW

**Review by Cheshiriks — COMMENTED** [review]
What about the 403 error for authorized users?

**Comment by sanyavertolet** [comment]
> What about the 403 error for authorized users?

I was unable to reproduce

File: api-gateway/src/main/kotlin/com/saveourtool/save/gateway/security/WebSecurityConfig.kt [hunk 1]
```diff
@@ -177,18 +185,25 @@ fun passwordEncoder(): PasswordEncoder = PasswordEncoderFactories.createDelegati
  * @param authorizationContext
  * @return [Mono] of [AuthorizationDecision]
  */
-private fun defaultAuthorizationDecision(
+private fun authorizationManagerAuthorizationDecision(
```
Comment (akuleshov7) [inline api-gateway/src/main/kotlin/com/saveourtool/save/gateway/security/WebSecurityConfig.kt:188]: can you please move it below method `defaultAuthorizationDecision`, where it is called ?

**Review by akuleshov7 — APPROVED** [review]

File: api-gateway/src/main/kotlin/com/saveourtool/save/gateway/security/WebSecurityConfig.kt (hunk 1 above)
Comment (sanyavertolet) [reply api-gateway/src/main/kotlin/com/saveourtool/save/gateway/security/WebSecurityConfig.kt]: Moved all dependant methods below `defaultAuthorizationDecision` and `userStatusBasedAuthorizationDecision`

**Pull request merged**